src/IO/S3/PocoHTTPClient.h

```diff
@@ -154,6 +159,16 @@ class PocoHTTPClient : public Aws::Http::HttpClient
     unsigned int s3_max_redirects;
     bool enable_s3_requests_logging;
     bool for_disk_s3;
+
+    /// Limits get request per second rate for GET, SELECT and all other requests, excluding throttled by put throttler
+    /// (i.e. throttles GetObject, HeadObject)
```

Comment:
btw can tests be made for this? May be a test which does a lot of requests per second and checks current metrics of get requests, etc? (not sure if there are current metrics, I remember only profile events, but CurrentMetrics can easily be added for testing)

Reply:
done